src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I do not understand why you are trying to argue that Stockfish is bad at antichess. Already more than half a year ago the results were very good (https://github.com/ddugovic/Stockfish/issues/110#issuecomment-261800148) and they still are (check the FICS link above). To me your arguments sound like a pretext. Why are you at war with SF antichess?